src/test/java/org/example/stepDefinition/fbLogin.java

```diff
@@ -20,12 +20,8 @@ public void userEnterValidatingText(){
         $(By.xpath("//input[@id='email']")).sendKeys("priyamunaga468@gmail.com");
         $(By.xpath("//input[@id='pass']")).sendKeys("Mokthika@143",Keys.ENTER);
         $(By.xpath("//span[contains(text(),'Priyanka Munaga')]")).should(Condition.exist);
-        sleep(7000);
```

Comment:
y did u do this

Reply:
pausing the code for such a long time is not recommended